driver/fuzz_target_runner.cpp

```diff
@@ -279,7 +279,7 @@ FuzzTargetRunner::~FuzzTargetRunner() {
 RunResult FuzzTargetRunner::Run(const uint8_t *data, const std::size_t size) {
   auto &env = jvm_.GetEnv();
   static std::size_t run_count = 0;
-  if (run_count < 2) {
+  if (run_count < 2 && FLAGS_hooks) {
```

Comment:
Can you clarify why is `FLAGS_hooks` checked here?

Reply:
This should have been checked all along: If `FLAGS_hooks` is set, we disable all instrumentation, including coverage. In particular, we should not interact with the CoverageMap in any, which the call below does.